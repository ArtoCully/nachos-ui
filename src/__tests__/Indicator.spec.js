import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'

import Indicator from '../Indicator'

it('should render correctly', () => {
  const View = require("View"); // eslint-disable-line
  const component = renderer.create(
    <Indicator position='left top' value='12'>
      <View />
    </Indicator>
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
