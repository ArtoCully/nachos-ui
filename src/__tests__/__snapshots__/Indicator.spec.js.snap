exports[`test should render correctly 1`] = `
<View
  style={undefined}>
  <View
    onLayout={[Function]} />
  <View
    style={
      Object {
        "left": 0,
        "position": "absolute",
        "top": 0,
      }
    }>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }>
      <View
        style={
          Array [
            Object {
              "backgroundColor": "#444",
              "borderRadius": 20,
              "padding": 10,
              "paddingVertical": 4,
            },
            Object {
              "backgroundColor": "#2f8cff",
            },
            Array [
              Object {
                "left": 0,
                "marginLeft": -18,
                "top": -10,
              },
            ],
          ]
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#fff",
                "fontSize": 12,
              },
              undefined,
            ]
          }>
          12
        </Text>
      </View>
    </View>
  </View>
</View>
`;
